cmake/modules/FindTAEF.cmake

```diff
@@ -41,18 +41,28 @@ set(TAEF_INCLUDE_DIRS ${TAEF_INCLUDE_DIR})
 set(TAEF_NUGET_BIN ${TAEF_INCLUDE_DIR}/../Binaries/Release)
 set(TAEF_SDK_BIN ${TAEF_INCLUDE_DIR}/../../Runtimes/TAEF)
 
+set(ARCH ${CMAKE_C_COMPILER_ARCHITECTURE_ID})
```

Comment:
The changes in this file seem unrelated to the rest of the changes in this PR. They should probably be in a separate PR.